faq/deploy-and-maintain-faq.md

```diff
@@ -245,6 +245,13 @@ Two reasons:
 - The two results are calculated in different ways. `information_schema.tables.data_length` is an estimated value by calculating the averaged length of each row, while the store size on the TiKV monitoring panel sums up the length of the data files (the SST files of RocksDB) in a single TiKV instance.
 - `information_schema.tables.data_length` is a logical value, while the store size is a physical value. The redundant data generated by multiple versions of the transaction is not included in the logical value, while the redundant data is compressed by TiKV in the physical value.
 
+#### Why does the transaction not have the Async Commit enabled or become a one-phase commit?
+
+In the following conditions, even you enable [Async Commit](/system-variables.md#tidb_enable_async_commit-new-in-v50) using the system variables and [one-phase commit](/system-variables.md#tidb_enable_1pc-new-in-v50), TiDB will not use the features:
+
+- If you enable TiDB Binlog, limited by the implementation principles of TiDB Binlog, TiDB will not have Async Commit or one-phase commit enabled to use the features.
```

Comment:
```suggestion
- If you have enabled TiDB Binlog, TiDB, restricted by the implementation principles of TiDB Binlog, does not use the Async Commit or one-phase commit feature.
```

Reply:
I cannot find the subject of the sentence "does not use the Async Commit or one-phase commit feature". And I think of course it is not an imperative sentence...

Reply:
updated

Reply:
And another bit. "Implementation principles" sounds a little weird to me. It's not something like a scientific law.
Maybe using"implementation mechanism" or just "limited by the implementation of .."?

Reply:
"principles" are removed.

Reply:
lgtm